src/engine.rs

```diff
@@ -141,28 +123,10 @@ impl FileEngineInner {
                 }
             }
 
-            // Only keep latest entries in cache, keep cache below limited size.
```

Comment:
Why remove this logic?

Reply:
It's already handled in `MemTable::append`.

Reply:
But the total cache size is not limited when start, this may cause loop OOM.